prime-router/metadata/fhir_mapping/hl7/datatypes/CX/Identifier.yml

```diff
@@ -0,0 +1,231 @@
+          valueOf: $cx42
+          expressionType: HL7Spec
+    extension_3:
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: cx-4-3-extension
+        valueString:
+          type: STRING
+          valueOf: $cx43
+          expressionType: HL7Spec
+
+type:
+  valueOf: datatype/CodeableConcept_var
+  expressionType: resource
+  vars:
+    code: $code | CX.5
+    system: $system
+
+extension_type:
+  generateList: true
+  expressionType: nested
+  condition: $cx5 NOT_NULL
+  vars:
+    cx5: CX.5
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: cx-5-extension
+    valueString:
+      type: STRING
+      valueOf: $cx5
+      expressionType: HL7Spec
+
+extension_assigningFacility:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: assigning-facility
+    valueReference:
+      valueOf: datatypes/HD/Organization
+      expressionType: reference
+      specs: CX.6
+
+period:
+  vars:
+    startDate: CX.7
+    endDate: CX.8
+  condition: $startDate NOT_NULL || $endDate NOT_NULL
+  expressionType: nested
+  expressionsMap:
+    start:
+      valueOf: "GeneralUtils.dateTimeWithZoneId(startDate,ZONEID)"
+      expressionType: JEXL
+    _start:
+      expressionType: nested
+      condition: $startDate NOT_NULL
+      expressionsMap:
+        extension_1:
+          generateList: true
+          expressionType: nested
+          expressionsMap:
+            url:
+              type: SYSTEM_URL
+              value: hl7v2-date-time
+            valueString:
+              type: STRING
+              valueOf: $startDate
+              expressionType: HL7Spec
+    end:
+      valueOf: "GeneralUtils.dateTimeWithZoneId(endDate,ZONEID)"
+      expressionType: JEXL
+    _end:
+      expressionType: nested
+      condition: $endDate NOT_NULL
+      expressionsMap:
+        extension_1:
+          generateList: true
+          expressionType: nested
+          expressionsMap:
+            url:
+              type: SYSTEM_URL
+              value: hl7v2-date-time
+            valueString:
+              type: STRING
+              valueOf: $endDate
+              expressionType: HL7Spec
+
+
+extension_4:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: assigning-jurisdiction
+    valueCodeableConcept:
+      valueOf: datatypes/CWE/CodeableConcept
+      expressionType: resource
+      specs: CX.9
+
+extension_5:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: assigning-agency-or-department
+    valueCodeableConcept:
+      valueOf: datatypes/CWE/CodeableConcept
+      expressionType: resource
+      specs: CX.10
+
+extension_6:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: identifier-index-name
```

Comment:
We can drop this in favor of continuing to use the `hl7-use` extension to be consistent